requirements_base.txt

```diff
@@ -173,3 +173,13 @@ ipaddress==1.0.18 \
 cached-property==1.3.1 \
     --hash=sha256:6562f0be134957547421dda11640e8cadfa7c23238fc4e0821ab69efdb1095f3 \
     --hash=sha256:fe045921fe75c873064028e9fbbe06121114ccf613227f4ba284fa7d4c9ff27f
+
+asset==0.6.12 \
+    --hash=sha256:addb25dbdbfa820c667e246f9c09122401df21cd9e94db919c72f8f04fc972eb
+passwordmeter==0.1.8 \
+    --hash=sha256:f4f7df54182072a137bca3e396f2dad354770ba596dd41f6646cd1f0665fee12
+aadict==0.2.3 \
+    --hash=sha256:a77328ac55dbb5735da99441870251befe135f687ab707a7a178561363b27704
+globre==0.1.5 \
+    --hash=sha256:ee214204f237e9114b8f61eeb61c2abd1e665ca3b59e5a6a0b070971c0bb12e2
+
```

Comment:
I've looked through these new dependencies, they appear to be okay (although their source code could use a good cleanup).

One issue is that `passwordmeter` includes [its own list of ten thousand common passwords](https://github.com/cadithealth/passwordmeter/tree/v0.1.8/passwordmeter/res), and [always loads it](https://github.com/cadithealth/passwordmeter/blob/v0.1.8/passwordmeter/__init__.py#L25-L26), even if we skip the `notword` factor (which we should do since it's redundant with the pwnedpasswords check). This wastes approximately 1.2MB of RAM (measured with [this `get_size` function](https://stackoverflow.com/a/40880923/2729778) on my laptop).